src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,61 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
 public class WorkWithFile {
+    static final int NAME_INDEX = 0;
+    static final int PARAM_INDEX = 1;
```

Comment:
```suggestion
    private static final int NAME_INDEX = 0;
    private static final int PARAM_INDEX = 1;
```
remember about access modifiers

Reply:
After changing I get a check style error:
https://mate-parttime.slack.com/archives/C027GNX3DFW/p1687456179766069